<!DOCTYPE HTML>
<!--
	Miniport by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>BIOINFORMATICS AND DEEP LEARNING</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="pics/mylogo.png">
	</head>
	<body>

		<!-- Nav -->
		<nav id="nav">
			<ul class="container">
					<li><a href="index.html"><font color="#808080">HOME</a></li>
					<li><a href="about.html"><font color="#808080">ABOUT</a></li>
				    <li><a href="research.html"><font color="#808080">RESEARCH</a></li>
					<li><a href="bioinformatics_and_deep_learning.html">BIOINFORMATICS AND DEEP LEARNING</a></li>
					<li><a href="resources.html"><font color="#808080">RESOURCES</a></li>
					<li><a href="contact.html"><font color="#808080">CONTACT</a></li>
			</ul>
		</nav>

		<!-- Publications -->
		<div class="container">
			<article id="BIOINFORMATICS AND DEEP LEARNING">
				    <br><br><br>
					<p style=align="left"><strong><font size="5.5">1. Construction of co-expression network of genes in M.oryzae</font><strong></p>
					<div class="row">

					<div class="3u 12u(mobile)">
					<p><img src="pics/wgcna.jpeg" alt="" height="240" width="240"></img></p>
					</div>
					<div class="9u 12u(mobile)">
					<p style="line-height: 30px;font-size:17px" align="left"><strong>Magnaporthe oryzae is a model fungus to study plant-microbe interaction. By leveraging internal and external transcriptomic datasets, I am creating a bulked matrix to generate a WGCNA co-expression network for this specie. This would guide us to elucidate how the fungus governs its initial invasive growth, and further rapid colonisation.<strong></p>
				    </div>
				  </div>

				  	<br>
				  	<p style=align="left"><strong><font size="5.5">2. Machine learning to predict proteins interaction</font><strong></p>
					<div class="row">
					<div class="8u 12u(mobile)">
					<p style="line-height: 30px;font-size:17px" align="left"><strong>We know a lot about proteins, especially about how they interact with each other and peptide sequences. Rather than manual experiments, we are able to predict interaction by inputting training data and generate a optimised way to explore unknown interactions and structural features. I am currently working on it with scikit-learn with python, and I am aiming to determine codon usage bias for many diverse systems.<strong></p>
				    </div>
					<div class="4u 12u(mobile)">
					<p><img src="pics/ML.png" alt="" height="261" width="290"></img></p>
					</div>

				  </div>

				  	<br>
				  	<p style=align="left"><strong><font size="5.5">3.Prediction of virulence genes in plant pathogens using Convolutional Neural Networks</font><strong></p>
					<div class="row">

					<div class="3u 12u(mobile)">
					<p><img  src="pics/deepL.jpeg" alt="" height="140" width="290"></img></p>
					</div>
					<div class="9u 12u(mobile)">
					<p style="line-height: 30px;font-size:17px" align="left"><strong>To effectively surveillance emerging threatens, we are delivering a method using TensorFlow to predict and classify pathogen genes using internal and external datasets. This facilitates us to be proactive when new isolates or lineages of pathogens come up, and react with solutions as soon as possible. <strong></p>
				    </div>
				  </div>
			</article>
		</div>




</body>
</html>
